packages/gui/src/components/app/AppProviders.tsx

```diff
@@ -1,14 +1,26 @@
 import React, { ReactNode, useEffect, useState, Suspense } from 'react';
 import { Provider } from 'react-redux';
 import { Outlet } from 'react-router-dom';
-import { useDarkMode, sleep, ThemeProvider, ModalDialogsProvider, ModalDialogs, LocaleProvider, LayoutLoading, dark, light, ErrorBoundary } from '@chia/core';
+import {
+  useDarkMode,
+  sleep,
+  ThemeProvider,
+  ModalDialogsProvider,
+  ModalDialogs,
+  LocaleProvider,
+  LayoutLoading,
+  dark,
+  light,
+  ErrorBoundary,
+} from '@chia/core';
 import { store, api } from '@chia/api-react';
 import { Trans } from '@lingui/macro';
 import { i18n, defaultLocale, locales } from '../../config/locales';
 import AppState from './AppState';
+import { WebSocket as WS } from 'ws';
```

Comment:
WebSocket is default import I think
import WebSocket from 'ws';

Reply:
I investigated it.

https://github.com/websockets/ws/blob/master/index.js#L10

```javascript
'use strict';

const WebSocket = require('./lib/websocket');

WebSocket.createWebSocketStream = require('./lib/stream');
WebSocket.Server = require('./lib/websocket-server');
WebSocket.Receiver = require('./lib/receiver');
WebSocket.Sender = require('./lib/sender');

WebSocket.WebSocket = WebSocket; // <-- Both default import and named import are correct
WebSocket.WebSocketServer = WebSocket.Server;

module.exports = WebSocket;
```

Reply:
Thanks, I will check our version and build